backend/tests/Tests/SqlCompiler.Tests.fs

```diff
@@ -143,17 +153,35 @@ let inlineWorksAtRoot =
       |> Ply.toTask
 
     let! expected = p "3 + 5"
-    let result = C.inline' "value" Map.empty expr
-    return Expect.equalExprIgnoringIDs result expected
+    let result =
+      uply {
+        let! result = C.inline' fns "value" Map.empty expr
+        return Expect.equalExprIgnoringIDs result expected
+      }
+    return (result |> Ply.toTask |> Async.AwaitTask |> Async.RunSynchronously)
```

Comment:
I am not happy with this solution, any suggestions please?
Perhaps one solution could be to make `equalExprIgnoringIDs` return a Ply<unit> ?